src/widgets/slideshowgeneratorwidget.cpp

```diff
@@ -66,6 +67,7 @@ SlideshowGeneratorWidget::SlideshowGeneratorWidget(Mlt::Playlist* clips, QWidget
     grid->addWidget(new QLabel(tr("Aspect ratio conversion")), 1, 0, Qt::AlignRight);
     m_aspectConversionCombo = new QComboBox();
     m_aspectConversionCombo->addItem(tr("Pad Black"));
+    m_aspectConversionCombo->addItem(tr("Pad Blur"));
```

Comment:
I want to be a little respectful to Linux distro packagers that will have not yet upgraded MLT (as it may not be released again soon) and hide this option if the `pillar_echo` filter is not available.

Reply:
> I want to be a little respectful to Linux distro packagers that will have not yet upgraded MLT (as it may not be released again soon) and hide this option if the `pillar_echo` filter is not available.

Done